### PR TITLE
Added workload repos table - pmem/benchmarking repo renaming required…

### PR DESCRIPTION
**Pull request by hubertsoltysiak**
… to work properly

---
This change is [Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/167)

### CODE REVIEW

**Review by KFilipek — COMMENTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/167)* status: all files reviewed, 5 unresolved discussions (waiting on @hubertsoltysiak)

**Review by hsoltysi — COMMENTED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/167)* status: all files reviewed, 5 unresolved discussions (waiting on @hubertsoltysiak)

**Comment by hubertsoltysiak** [comment]
Adding rendered preview:
image

**Comment by lukaszstolarczuk** [comment]
you can see here some weird part, I was afraid something like this could happen:
image
// taken from https://github.com/hubertsoltysiak/pmem.github.io/blob/master/repoindex.md

hint, this is not the real preview, it's not fully rendered (like Jekyll will render it). As Piotr suggested, you could enable it to be visible e.g. here: https://hubertsoltysiak.github.io (just need to turn it on, it in your hubertsoltysiak/pmem.github.io repo settings).

**Comment by hubertsoltysiak** [comment]
> you can see here some weird part, I was afraid something like this could happen

This "weird" content is connected with .md file and is also visible in original repo.
When rendered - page looks good (check at https://hubertsoltysiak.github.io/repoindex) :
image

File: repoindex.md [hunk 1]
```diff
@@ -261,6 +261,34 @@ organization in GitHub under
 		</tr>
 	</table>
 </fieldset>
+<fieldset class="repo">
+	<legend>
+		pmem-related workloads
+	</legend>
+	<p>
+	These repos contain non-volatile memory related workloads.
```
Comment (pbalcer) [inline repoindex.md:269]: Persistent Memory workloads?

File: repoindex.md [hunk 2]
```diff
@@ -261,6 +261,34 @@ organization in GitHub under
...
+	Their concept is based on combined market ready solutions with
```
Comment (pbalcer) [inline repoindex.md:270]: They are meant to be production-ready solutions for PMem.

File: repoindex.md [hunk 3]
```diff
@@ -261,6 +261,34 @@ organization in GitHub under
...
+	persistent memory. Workloads are delivered in SaaC (Solution as
```
Comment (pbalcer) [inline repoindex.md:271]: Is the acronym "SaaC" a thing or our own invention? I googled a bit and couldn't find anything relevant.
If it's our own invention then I'd drop it.

File: repoindex.md [hunk 4]
```diff
@@ -261,6 +261,34 @@ organization in GitHub under
...
+	a Code) concept, in the form of bash/ansible scripts operating
+	on the provided infrastructure. The infrastructure requirements are
+	described in the documentation (README.md). They use multiplication
```
Comment (pbalcer) [inline repoindex.md:274]: I don't understand what this last sentence means. What is multiplication in this context?

File: repoindex.md [hunk 5]
```diff
@@ -261,6 +261,34 @@ organization in GitHub under
...
+	based on virtualization, contenerization or clustering.
+	<p>
+	<table>
+		<tr>
+			<th>Repo Name</td>
+			<th>Description</th>
+		</tr>
+		<tr>
+			<td><a href="https://github.com/pmem/kvm-redis">kvm-redis</a></td>
+			<td>Workload using memtier_benchmark @ client: VMs and Redis database @ server: VMs</td>
```
Comment (pbalcer) [inline repoindex.md:284]: In the intro you say that these are meant to be "market ready solutions". But here both the workloads are just benchmarks?

**Review by lukaszstolarczuk — COMMENTED** [review]
och, thanks, that's great then :+1: 

> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/167#-:-MnfLO8h2Wdhk6GbB9zP:b-pvb0al)* status: all files reviewed, 5 unresolved discussions (waiting on @hubertsoltysiak)

**Review by KFilipek — COMMENTED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/167)* status: all files reviewed, 5 unresolved discussions (waiting on @hubertsoltysiak)

File: repoindex.md (hunk 1 above)
Comment (hubertsoltysiak) [reply repoindex.md:269]: ack

File: repoindex.md (hunk 2 above)
Comment (hubertsoltysiak) [reply repoindex.md:270]: updated

File: repoindex.md (hunk 3 above)
Comment (hubertsoltysiak) [reply repoindex.md:271]: dropped

File: repoindex.md (hunk 4 above)
Comment (hubertsoltysiak) [reply repoindex.md:274]: also dropped

File: repoindex.md (hunk 5 above)
Comment (hubertsoltysiak) [reply repoindex.md:284]: Marked ready solution is redis database instances on kubernetes cluster or virtualised with kvm. Benchmark (memtier_benchmark) is used as artificial source of traffic to load database instances with requests.

**Comment by hubertsoltysiak** [comment]
Updated visualization:
image

**Pull request merged**